specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -2185,22 +2185,12 @@
               "$ref": "#/definitions/RuleSet"
             }
           },
-          "202": {
-            "description": "Accepted. The request has been accepted for processing and the operation will complete asynchronously.",
-            "schema": {
-              "$ref": "#/definitions/RuleSet"
-            }
-          },
           "default": {
             "description": "CDN error response describing why the operation failed.",
             "schema": {
               "$ref": "#/definitions/AfdErrorResponse"
             }
           }
-        },
-        "x-ms-long-running-operation": true,
-        "x-ms-long-running-operation-options": {
-          "final-state-via": "azure-async-operation"
         }
```

Comment:
Yes that is correct. The DELETE operation is still LRO because it will need to delete resources and that can take some time. PUT does not create resources per se, it is simply a container.